src/java/org/apache/cassandra/net/OutboundConnection.java

```diff
@@ -824,7 +825,11 @@ boolean doRun(Established established)
                         Message.serializer.serialize(next, out, messagingVersion);
 
                         if (sending.length() != sendingBytes + messageSize)
+                        {
+                            DataOutputBufferFixed out2 = new DataOutputBufferFixed(ByteBuffer.allocate(sendingBytes + messageSize));
+                            Message.serializer.serialize(next, out2, messagingVersion);
```

Comment:
why? this buffer gets ignored

Reply:
Oh this was debug logic I failed to remove. Thanks